packages/docs/docs/docs/architecture.md

```diff
@@ -0,0 +1,97 @@
+---
+id: architecture
+title: On-chain Architecture
+---
+
+ZeppelinOS's features such as upgradeability, EVM package linkage, etc. can all be used independently as standalone components. However, they work best together when used within a full ZeppelinOS project. This can be done using the command:
+
+TODO: what is the command?
+```
+zos init --full
```

Comment:
Note that upgradeability and linking packages can be done also on lightweight apps (ie --full is not required)